README.md

```diff
@@ -80,10 +80,10 @@ With the following command, you can easily analyze the Android sample and output
 See our demo [here](https://pulorsok.github.io/ruleviewer/web-report-demo).
 
 ```bash
-quark -a sample.apk -s -w quark_report.html
```

Comment:
By default, Quark uses all built-in rules to run the analysis, so we could drop the option, `all_rules`. (Tested with click 8.1.3)

Reply:
The fix is done.